app/src/components/Profile/DeleteAccountModal.tsx

```
@@ -22,39 +24,55 @@ const DeleteAccountModal: React.FC<Props> = ({
+              <label
+                htmlFor="email"
+                className="font-bold text-[12px] leading-[15px] tracking-[2px] text-white-blue mix-blend-normal opacity-90 block mt-[19px] uppercase sm:text-[11px]"
               >
-                {t('Profile.text.please_enter')}
-              </Trans>
-            </label>
-            <Input type="email" id="email" name="email" ref={register} />
-            {errors?.email?.message && <ErrorText message={String(t(errors.email.message))} />}
+                <Trans
+                  components={[
+                    <div className="text-sapphire-blue font-bold" />,
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)